Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bevy_local_commands"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
I think this could qualify as breaking change, as the state management has changed and maybe changes are applied in a different order?

Perhaps it's safer to bump to 0.7, just to be safe.

Reply:
Fair enough!